src/main/java/com/github/curiousoddman/rgxgen/RgxGen.java

```diff
@@ -17,11 +17,11 @@
 /* **************************************************************************/
```

Comment:
TODO:
1. Correct README - can set properties as int
2. FIXME and TODO (or create separate task)
3. Cleanup - probably not all methods are used now.